lib/Onyx.js

```diff
@@ -54,7 +55,8 @@ const deferredInitTask = createDeferredTask();
  * @param {Object} [withOnyxInstanceState]
  * @returns {Mixed}
  */
-const getSubsetOfData = (sourceData, selector, withOnyxInstanceState) => selector(sourceData, withOnyxInstanceState);
+const getSubsetOfData = (sourceData, selector, withOnyxInstanceState) =>
+    selector(sourceData, withOnyxInstanceState);
```

Comment:
I also question if updates like this adding new lines for lines that weren't already that long are necessary. They're adding a lot of noise to this PR IMO.

Reply:
I cleaned all the formatting changes, should be a lot let noisy now